otelib/transformation.py

```diff
@@ -0,0 +1,42 @@
+import json
+import requests
+
+from oteapi.models import TransformationConfig
+
+from .apierror import ApiError
+from .abstractfilter import AbstractFilter
+from .settings import Settings
+
+
+class Transformation(AbstractFilter):
+    """Context class for the Transformation Strategy Interfaces"""
+
+    def __init__(self, url):
+        self.url = url
+        self.settings = Settings()
```

Comment:
Maybe this should be a part of the `AbstractFilter` init method, which can then be referenced using `super().__init__(url)`?
This is a comment for all of these classes (sub-classes of `AbstractFilter`).

Reply:
I was thinking the same, but then `__init__()` cannot be annotated as an @abstractmethod in `AbstractFilter`. But that might be fine? In that case the filters not even need to define an `__init__()` method.

Reply:
It still can. It can both be an abstract method _and_ have default content. Or you mean `super()` cannot be called on an abstract method? Hmm...

But yes. Having the filters not even defining an `__init__` method would solve that. Keep it abstract and keep it with a default. And also it would be much cleaner. We should, however, make sure that it can still be overwritten, while also calling back to it with `super().__init__()`.